scan/src/subscriptions/BlockSub.re

```diff
@@ -0,0 +1,122 @@
+type aggregate_t = {count: int};
+
+type transactions_aggregate_t = {aggregate: option(aggregate_t)};
+
+type internal_t = {
+  height: int,
+  hash: Hash.t,
+  proposer: Address.t,
+  timestamp: MomentRe.Moment.t,
+  transactions_aggregate: transactions_aggregate_t,
+};
+
+type t = {
+  height: int,
+  hash: Hash.t,
+  proposer: Address.t,
+  timestamp: MomentRe.Moment.t,
+  txn: int,
+};
+
+let toExternal = ({height, hash, proposer, timestamp, transactions_aggregate}) => {
+  height,
+  hash,
+  proposer,
+  timestamp,
+  txn:
+    switch (transactions_aggregate.aggregate) {
+    | Some(aggregate) => aggregate.count
+    | _ => 0
+    },
+};
+
+module MultiConfig = [%graphql
+  {|
+  subscription Blocks($limit: Int!, $offset: Int!) {
+    blocks(limit: $limit, offset: $offset, order_by: {height: desc}) @bsRecord {
+      height @bsDecoder(fn: "GraphQLParser.int64")
+      hash: block_hash @bsDecoder(fn: "GraphQLParser.hash")
+      proposer @bsDecoder(fn: "Address.fromHex")
```

Comment:
Don't think this is the right thing to do. @taobun can you confirm?

Reply:
In database we store consensus validator address, so if you want validator address and moniker just query more data
```
validator{
      consensus_address
      operator_address
      moniker
    }
```